guides/src/content/user/shipments/calculators.md

```diff
@@ -69,7 +69,7 @@ A customer whose order subtotal equals $35 would be offered a shipping cost of $
 
 ### Custom Calculators
 
-You can define your own calculator if you have more complex needs. In that case, check out the [Calculators Guide](../developer/calculators.html).
```

Comment:
@KajZukowski this should link to https://guides.spreecommerce.org/developer/core/calculators.html

`../developer/core/calculators.html`

Reply:
Addressed in the new commit